kite-data/kite-data-crunch/pom.xml

```diff
@@ -318,5 +318,36 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hdp2-release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-deploy-plugin</artifactId>
```

Comment:
I'll add a comment. The issue is that we only want to deploy the classified version of jars during the HDP build, not the primary artifacts. The only way to do that is to disable deploy of main artifacts which is done in the parent POM and then to add this rule to deploy a specific file.